analysis/cross_partition_combiners.py

```diff
@@ -94,3 +91,15 @@ def _sum_metrics_to_metric_utility(
         ratio_data_dropped=data_dropped,
         absolute_error=absolute_error,
         relative_error=relative_error)
+
+
+def _partition_selection_per_to_cross_partition(
+        prob_keep: float) -> metrics.PrivatePartitionSelectionUtility:
+    """Creates cross-partition partition selection utility from 1 partition utility."""
```

Comment:
The name `PrivatePartitionSelectionUtility` sounds odd, should we rename them to `PrivatePartitionSelectionMetrics` instead?

Also, what does `from 1 partition utility` mean in the docstring? It seems to create a utility from 1 partition keep probability?

Reply:
Agree, `PrivatePartitionSelectionMetrics` is better

Fixed the docstring